pkg/generate/code/set_sdk.go

```diff
@@ -190,6 +190,22 @@ func SetSDK(
 			}
 		}
 
+		fc, ok := fieldConfigs[memberName]
+		if ok && fc.IsSecret {
```

Comment:
something to address in a future PR, but it would be nice to have a code comment here similar to below on line 210 that shows what the generated Go code looks like from this section.